projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -1516,7 +1516,7 @@ function wpcom_launchpad_request_user_attributes( $attributes, $client_wrapper =
 	}
 
 	$user_attributes = get_object_vars( $decoded_body->user_attributes );
-	if ( ! is_array( $user_attributes ) ) {
+	if ( ! $user_attributes ) {
```

Comment:
This is a behavior change. Before an empty array would result in no error, now it will.